src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -13,7 +13,6 @@
     <CLSCompliant>false</CLSCompliant>
     <!-- rev'ed to 1.4.1 so that we can drop pre-release down to beta -->
```

Comment:
orthogonal but it looks like this comment and the definecontants are dead code now.